### ISSUE TITLE
can't finetune model to convergence

### ISSUE THREAD

**Issue by Mordoal** [maybeLx/MVSFormerPlusPlus#37]
My datasets is consist of pics with the same size as dtu dataset, while depth range from like 30mm to 50mm. Changing lr to e-5, vir_le to e-7, min_lr to e-4 are all the same. Loss never decay, and stays at 9.7. And the result cannot be said 'good'. Could you please give me some advice on how I can improve the model's performance on my custom dataset?

**Comment by maybeLx** [comment]
You need to adjust the depth range. Additionally, you can load our checkpoint and fine-tune it with your own dataset, ensuring that you follow the DTU dataset structure. Also, please note that the camera pose is in w2c format, not c2w, and follows the OpenCV coordinate system.
